common/automation-scripts/OrdersCreation/purchase-order-data-load.sh

```diff
@@ -0,0 +1,172 @@
+BEGIN
+    for line_counter in 1..polines_per_order
+        loop
+            INSERT INTO ${TENANT}_mod_orders_storage.po_line (id, jsonb)
+            VALUES (public.uuid_generate_v4(),
+                       -- add other fields to increase processing complexity
+                    jsonb_build_object('id', public.uuid_generate_v4(),
+                                       'acquisitionMethod', 'df26d81b-9d63-4ff8-bf41-49bf75cfa70e',
+                                       'rush', false,
+                                       'cost', json_build_object(
+                                               'currency', 'USD',
+                                               'discountType', 'percentage',
+                                               'listUnitPrice', 1,
+                                               'quantityPhysical', 1,
+                                               'poLineEstimatedPrice', 1
+                                           ),
+                                       'alerts', json_build_array(),
+                                       'source', 'User',
+                                       'physical', jsonb_build_object('createInventory', 'None'),
+                                       'isPackage', false,
+                                       'orderFormat', 'Physical Resource',
+                                       'vendorDetail', jsonb_build_object('vendorAccount', '${AccNum}'),---------------------------------
+                                       'titleOrPackage', 'ABA Journal',
+                                       'automaticExport', true,
+                                       'publicationDate', '1915-1983',
+                                       'purchaseOrderId', order_id,
+                                       'poLineNumber', concat(ponumber, '-', line_counter),
+                                       'claims', json_build_array(),
+                                       'metadata', jsonb_build_object(
+                                               'createdDate', '2018-07-19T00:00:00.000+0000',
+                                               'createdByUserId', '28d1057c-d137-11e8-a8d5-f2801f1b9fd1',
+                                               'updatedDate', '2018-07-19T00:00:00.000+0000',
+                                               'updatedByUserId', '28d1057c-d137-11e8-a8d5-f2801f1b9fd1'
+                                           )
+                        ));
+        end loop;
+END
+\$\$ LANGUAGE plpgsql;
+
+
+-- CREATE sample data
+-- 1 - amount of organizations to be created
+-- 2 - amount of orders per organization
+-- 3 - amount of polines per order
+select public.generate_data_for_edifact_export(1, ${orderesNum}, 1);
+
+
+--check
+--SELECT count() FROM ${TENANT}_mod_organizations_storage.organizations WHERE jsonb ->> 'code' LIKE 'PERF_TEST_ORG%';
+--SELECT count(*) FROM ${TENANT}_mod_orders_storage.purchase_order;
+SELECT count(*)
+FROM ${TENANT}_mod_orders_storage.po_line;"
+
```

Comment:
What is the significance of having 5 white lines here? Can some or all be removed?